akka-actor-typed-tests/src/test/scala/docs/akka/typed/InteractionPatternsSpec.scala

```diff
@@ -82,11 +82,19 @@ class InteractionPatternsSpec extends ScalaTestWithActorTestKit with AnyWordSpec
       val context = new {
         def self = probe.ref
```

Comment:
I will see I I can remove this hack and still having something to test the documentation sample, but not a priority.

Reply:
Yeah, not important to do anything about IMO

Reply:
Well, I had a look at it. The code is lying. It pretends to send a `Response` to `context.self`, but `Response` is not part of `CookieFabric`'s protocol. And it compiles because it's the probe, not the real self. 

I will send another PR to fix this.